app/scripts/templates/connect_another_device.mustache

```diff
@@ -4,13 +4,18 @@
     <div class="success success-authenticated visible">{{#t}}This Firefox is connected{{/t}}</div>
   {{/isSignedIn}}
   {{^isSignedIn}}
-    <div class="success success-not-authenticated visible">{{#t}}Email verified{{/t}}</div>
+    {{#isSignUp}}
+      <div class="success success-not-authenticated visible">{{#t}}Email verified{{/t}}</div>
+    {{/isSignUp}}
+    {{#isSignIn}}
+      <div class="success success-not-authenticated visible">{{#t}}Signin confirmed{{/t}}</div>
+    {{/isSignIn}}
   {{/isSignedIn}}
   <section>
```

Comment:
nit: maybe you want to just add a class to this section, something like `cad-instructions` and then css it with `.cad instructions p`.  Nit though!

Reply:
>  maybe you want to just add a class to this section, something like cad-instructions and then css it with .cad instructions p. Nit though!

You know... I originally added the class to add some styling. Turns out I didn't add any styling. Removing the extra class, avoiding the naming problem all together!